NodeMCU/src/Client/client.ino

```diff
@@ -11,8 +11,20 @@ void setup() {
 void loop() {
  if(WiFi.status() == WL_CONNECTED){                      //Check WiFi connection status
    HTTPClient http;                                      //Declare object of class HTTPClient
+   HTTPClient httpHat;
```

Comment:
Note: Clients will mostly ever need to GET two different links from the server, which greatly reduces output time.